tests/unit/acl.tcl

```diff
@@ -217,6 +217,20 @@ start_server {tags {"acl"}} {
         set e
     } {*NOPERM*}
 
+    test {ACLs set can include subcommands, if already full command exists} {
+            r ACL setuser bob +pfcount +pfcount|hll
+            # Validate the commands have got engulfed to +pfcount.
+            set cmdstr [dict get [r ACL getuser bob] commands]
+            assert_match {-@all +pfcount} $cmdstr
```

Comment:
This is deceptively passing, because "-@all +pfcount|hll" will actually match here.

Reply:
Well the response was correct `-@ALL +pfcount`, I had verified it separately. `ACL LIST` handles it correctly.